shell/server/core.js

```diff
@@ -32,13 +32,17 @@ class SandstormCoreImpl {
     const _this = this;
     return inMeteor(() => {
       const hashedSturdyRef = hashSturdyRef(sturdyRef);
-      const token = ApiTokens.findOne(hashedSturdyRef);
-      const requirements = [
-        {
-          tokenValid: hashedSturdyRef,
```

Comment:
I'm not dropping the requirement. It is more natural to treat this requirement as implied rather than explicitly list it. The requirements in this list will be explicitly checked during restoreInternal(), but checking the parent token there would be redundant since restoreInternal() is already going to follow parent tokens in order to figure out what it's restoring, and will already fail if any of them are no longer valid. I've changed comments in supervisor.capnp to indicate that the parent token requirement is implied and won't be listed explicitly. Since the parent token is passed into Supervisor.restore() as a separate argument, it doesn't need to be listed again under `requirements`.